op-proposer/proposer/driver.go

```diff
@@ -303,18 +305,52 @@ func (l *L2OutputSubmitter) FetchOutput(ctx context.Context, block *big.Int) (*e
 }
 
 // ProposeL2OutputTxData creates the transaction data for the ProposeL2Output function
-func (l *L2OutputSubmitter) ProposeL2OutputTxData(output *eth.OutputResponse) ([]byte, error) {
-	return proposeL2OutputTxData(l.l2ooABI, output)
+func (l *L2OutputSubmitter) ProposeL2OutputTxData(output *eth.OutputResponse, consumerId string) ([]byte, error) {
```

Comment:
How about we invoke `ConstructConsumerID` inside this function so that the function does not need to take `consumerId` as input? Each OP proposer will only have 1 unique consumer ID anyway

Reply:
That makes sense.